moto/dynamodb2/models.py

```diff
@@ -1123,6 +1126,14 @@ def _trim_results(self, results, limit, exclusive_start_key, scanned_index=None)
                     break
 
         last_evaluated_key = None
+        size_limit = 1000000  # DynamoDB has a 1MB size limit
+        item_size = sum([res.size() for res in results])
```

Comment:
Same here as previous nit; can drop the square brackets.
```suggestion
        item_size = sum(res.size() for res in results)
```